000-vip-init.php

```diff
@@ -92,13 +92,24 @@
 
 // Support a limited number of additional "Internal Events" in Cron Control.
 // These events run regardless of the number of pending events, and they cannot be deleted.
-define( 'CRON_CONTROL_ADDITIONAL_INTERNAL_EVENTS', array(
+$internal_cron_events = array(
 	array(
 		'schedule' => 'hourly',
 		'action'   => 'wpcom_vip_support_remove_user_via_cron', // Automattic\VIP\Support_User\User::CRON_ACTION
 		'callback' => array( 'Automattic\VIP\Support_User\User', 'do_cron_cleanup' ),
-	),
-) );
+	)
+);
+
+if ( defined( 'WPCOM_VIP_RUN_CONNECTION_PILOT' ) && true === WPCOM_VIP_RUN_CONNECTION_PILOT ) {
+	$internal_cron_events[] = array(
+		'schedule'  => 'hourly',
```

Comment:
Possibly. Ideally we dig into the disconnection issues to find the underlining cause. If possible, we could then try to prevent the issue in the first place.

If sites are disconnecting often, then I'm not sure if the best course of action is to try and fix every few minutes.